app/views/pages/get-started.liquid

```diff
@@ -16,12 +16,12 @@ Besides standard web technologies like HTML, CSS, JavaScript, platformOS uses a
 
 - [Command line crash course](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Understanding_client-side_tools/Command_line)
 
-**GIT** knowledge can be very basic. You might be needed to clone a repository, but for the most platformOS use cases we’ve developed a friendly command line tool that will help with everyday tasks.
+You'll need **GIT** to access the example code from this tutorial on GitHub. It's also recommended to use a version control system to track and manage code changes during your daily work.
```

Comment:
Opinionated.

Reply:
To be precise, anyone can access public repositories on GitHub and download the code - git, or even an account in GitHub, is not required :) I like your suggested wording @diana-lakatos